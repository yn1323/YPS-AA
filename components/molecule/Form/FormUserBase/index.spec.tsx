import React from 'react'
import stories from './index.stories'
import Component, { Props } from '.'
import { mountWithTheme } from '@test/TestWrapper'

describe(`${Component.name}`, () => {
  const component = mountWithTheme(<Component {...(stories.args as Props)} />)
  test(`${Component.name}-snapshot`, () => {
    expect(component).toMatchSnapshot()
  })
})
