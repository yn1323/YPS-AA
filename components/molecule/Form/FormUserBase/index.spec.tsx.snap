// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`molecule/Form/UserBase molecule/Form/UserBase-snapshot 1`] = `
<FormUserBase
  definition="User type"
  item={
    <div>
      aaa
    </div>
  }
>
  <FormUserBase__Container>
    <div
      className="FormUserBase__Container-sc-143288g-0 kqhLZP"
    >
      <FormUserBase__TitleWrapper>
        <div
          className="FormUserBase__TitleWrapper-sc-143288g-1 jSzBJn"
        >
          <FormUserBase__Title
            $position="center"
          >
            <div
              className="FormUserBase__Title-sc-143288g-2 dKTgsS"
            >
              User type
            </div>
          </FormUserBase__Title>
        </div>
      </FormUserBase__TitleWrapper>
      <div>
        aaa
      </div>
    </div>
  </FormUserBase__Container>
</FormUserBase>
`;
